drivers/overlay/ov_network.go

```diff
@@ -760,58 +757,38 @@ func (n *network) watchMiss(nlSock *nl.NetlinkSocket) {
 				continue
 			}
 
-			logrus.Debugf("miss notification: dest IP %v, dest MAC %v", ip, mac)
```

Comment:
miss notifications were a handy signature of ipvs connection tracking timeouts, since the after-timeout connection attempt arps for the VIP.  i miss them.  when is `time.Since(t)` expected to exceed a second?

Reply:
time.Since is simple used as a rate limiter below, in order not to flood the logs with a notification that potentially is driven by data packets.
Anyway just as an heads up, the whole watchMiss logic is gone from swarm mode, so nothing of this code will be executed

Reply:
ah. that would explain the lack of messages then.

Reply:
have `l2miss` and `l3miss` been disabled on `vxlan0`?  No reason to generate the notification if it's not being consumed.

Reply:
actually, for my diagnostic purposes, there might be :-).

`ip -t monitor neigh` in the overlay namespace can provide a record of miss notifications we used to get from the daemon.

Reply:
actually the whole go routine is neither spawned.
this will save per overlay namespace: 1 thread (that was used for the recv syscall) and 1 netlink socket. Definitely too many resources only to print a log :)

Reply:
reference PR: https://github.com/docker/libnetwork/pull/2047